### PR TITLE
Added header design for a Become a parner page

### PR DESCRIPTION
**Pull request by Alen-Kalac**
Added initial design for header and rough design of a body for a "Become a partner page".

### CODE REVIEW

**Review by edinmavric — CHANGES_REQUESTED** [review]
Use scss nesting logic, that's why we use scss, and use react-router-dom in App, revert index.css from app.css, and rename them to scss files to, i think that header should be outside of the Form too

File: partners/src/components/header/style.scss
Comment (edinmavric) [inline partners/src/components/header/style.scss:1]: Kill's the point of implementing scss

File: partners/src/components/header/index.tsx
Comment (edinmavric) [inline partners/src/components/header/index.tsx:1]: We don't know if this is header of Form, or header of Page?

File: partners/src/index.css
Comment (edinmavric) [inline partners/src/index.css:1]: Delete content from App.scss, and store it here, rename it to index.scss too

File: partners/src/App.css
Comment (edinmavric) [inline partners/src/App.css:1]: Delete content

File: partners/src/App.tsx
Comment (edinmavric) [inline partners/src/App.tsx:1]: implement react-router-dom with Footer and Header outside out of routes

**Review by edinmavric — CHANGES_REQUESTED** [review]

**Comment by hamzaplojovic** [comment]
Amazing work, just like we requested. We talked about the font, I am sure you will implement that very soon. As for the inputs: `business name`, `email`, safe `password` input with password strength check, `phone number`, textarea input for `description`, and a file input for the `logo`. For now, these are the ones we will be using, and we can add more inputs as organization schema scales to more features, like input for certifications, staff members, open jobs, and business locations around the town, but for now, these highlighted ones are enough. Thank you!

**Review by hamzaplojovic — CHANGES_REQUESTED** [review]
Great job, just fill out the content of the page with the form with inputs we requested in the comment

**Review by hamzaplojovic — APPROVED** [review]
Great job!

**Pull request merged**